entities/build.gradle.kts

```diff
@@ -0,0 +1,15 @@
+repositories {
+    mavenCentral()
+    maven {
+        url = uri("https://repo.akka.io/maven")
+    }
+}
```

Comment:
```suggestion

```

I think these are useless since they are already declared in the build file of the root project.